ws-android/files/workspace/a01-create-ant-android-apk.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash  
```

Comment:
We should probably give an example app, and add a runner to be able to build it with the press of a button.